### PR TITLE
feat: add dashboard seed and tests for scheduled delivery edge cases

### PR DESCRIPTION
**Pull request by notgiorgi**
### Description:

Added a new seed file for testing scheduled delivery edge cases and enhanced the minimal E2E tests to verify screenshot ready indicators.

The seed creates a dashboard with various edge cases:

- A bar chart with date filter that returns data
- A table with a filter that returns no results
- A table with data but an invalid metric
- An orphaned chart tile (chart deleted but tile remains)

Added a new E2E test that verifies the screenshot ready indicator works correctly with these edge cases, checking that it properly reports total tiles, errored tiles, and completion status.

### CODE REVIEW

**Comment by notgiorgi** [comment]
* **#19472** <a href="https://app.graphite.com/github/pr/lightdash/lightdash/19472?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/lightdash/lightdash/19472?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#19471** <a href="https://app.graphite.com/github/pr/lightdash/lightdash/19471?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/e2e/cypress/e2e/app/minimal.cy.ts [hunk 1]
```diff
@@ -91,4 +91,40 @@ describe('Minimal pages', () => {
...
+                expect(Number(errored)).to.be.greaterThan(0);
+            });
+
+        // Verify status is completed-with-errors (due to orphan/error tiles)
+        cy.get(`#${SCREENSHOT_READY_INDICATOR_ID}`).should(
+            'have.attr',
+            'data-status',
+            'completed-with-errors',
+        );
+    });
```
Comment (notgiorgi) [inline packages/e2e/cypress/e2e/app/minimal.cy.ts:129]: TODO :: we need to enable screenshot indicator flag for previews

**Comment by notgiorgi** [comment]
### Merge activity

* **Jan 21, 12:45 PM UTC**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.com/github/pr/lightdash/lightdash/19472).
* **Jan 21, 12:45 PM UTC**: @notgiorgi merged this pull request with [Graphite](https://app.graphite.com/github/pr/lightdash/lightdash/19472).
